framework/src/Volo.Abp.AspNetCore.Mvc.Client/Volo/Abp/AspNetCore/Mvc/Client/AbpAspNetCoreMvcClientModule.cs

```diff
@@ -9,5 +11,16 @@ namespace Volo.Abp.AspNetCore.Mvc.Client;
     )]
 public class AbpAspNetCoreMvcClientModule : AbpModule
 {
-    
+    public override void ConfigureServices(ServiceConfigurationContext context)
+    {
+        var abpHostEnvironment = context.Services.GetAbpHostEnvironment();
+        if (abpHostEnvironment.IsDevelopment())
+        {
+            Configure<AbpAspNetCoreMvcClientCacheOptions>(options =>
+            {
+                options.TenantConfigurationCacheAbsoluteExpiration = TimeSpan.Zero;
+                options.ApplicationConfigurationDtoCacheAbsoluteExpiration = TimeSpan.Zero;
```

Comment:
Instead of zero, we may set a value like 5 seconds. Otherwise, it will make request to server too frequently, even multiple times in a single HTTP request.